src/index.js

```diff
@@ -164,4 +156,9 @@ function noUndef(path) {
   return true;
 }
 
+function removeUndef(o) {
```

Comment:
Since this method removes both `null` and `undefined`, the name `removeUndef` seems misleading. Maybe something more like `removeUndefOrNull` or something.